pyispyb/core/routes/webservices/userportalsync.py

```diff
@@ -0,0 +1,48 @@
+import logging
+from fastapi import HTTPException, status, Depends
+from fastapi.responses import Response, JSONResponse
+from pyispyb.app.base import AuthenticatedAPIRouter
+from ...modules import userportalsync as crud
+from ...schemas import userportalsync as schema
+from ....dependencies import permission
+
+
+logger = logging.getLogger("ispyb")
+
+router = AuthenticatedAPIRouter(prefix="/userportalsync", tags=["User Portal Sync"])
```

Comment:
Use the base router you defined, instead of defining a new one here:
![Screenshot 2022-08-26 at 16 48 44](https://user-images.githubusercontent.com/4463399/186931815-87d6282c-0ff5-4f06-b3ca-8d770057c35d.png)

Reply:
Done! 

https://github.com/ispyb/py-ispyb/pull/210/commits/820d8bd71b5251308e0e1a84c326736b211dda71